tests/route_tester/base.py

```diff
@@ -16,6 +17,10 @@ def __init__(self, test, test_client):
     def endpoint(self):
         if not self._endpoint:
             self._endpoint = self.route.format(**self.params)
+            # This is specifically for the search route but I cannot see any draw
+            # backs with having this here.
+            if re.search("//", self._endpoint):
+                self._endpoint = re.sub("//", "/", self._endpoint)
```

Comment:
Does this do what you want?

https://docs.python.org/2/library/os.path.html#os.path.normpath